pkg/apiserver/api_installer.go

```diff
@@ -812,12 +812,14 @@ func typeToJSON(typeName string) string {
 		return "boolean"
 	case "uint8", "int", "int32", "int64", "uint32", "uint64":
 		return "integer"
-	case "byte":
-		return "string"
 	case "float64", "float32":
 		return "number"
+	case "byte":
+		return "string"
 	case "util.Time":
 		return "string"
+	case "[]string":
```

Comment:
This seems wrong to me. At minimum, there would need to be a comment explaining why this is doing this.